controller/search_blackbox_test.go

```diff
@@ -63,6 +66,135 @@ func (s *searchControllerTestSuite) SetupTest() {
 	s.controller = NewSearchController(s.svc, s.GormDB, spaceBlackBoxTestConfiguration)
 }
 
+func (s *searchControllerTestSuite) TestSearchWorkItemsCSV() {
+	deserialize := func(csvStr string) ([]map[string]string, error) {
+		if csvStr == "" || csvStr == "\n" {
+			return []map[string]string{}, nil
+		}
+		var result []map[string]string
+		reader := csv.NewReader(strings.NewReader(csvStr))
```

Comment:
>Since this is a test, doesn't it make sense to configure the CSV reader to have a fixed FieldsPerRecord? (@jarifibrahim thoughts?)

@kwk I don't really understand what do you mean by `fixed FieldsPerRecord`. Do you mean fixed number of columns?

Reply:
Fixed e5726d5. I set the `FieldsPerRecord` to `0` now. This way, the parser checks if the subsequent rows match the number of fields of the first (title) row.

Reply:
@michaelkleinhenz you can undo this change because of me misreading the documentation: https://github.com/fabric8-services/fabric8-wit/pull/2347#commitcomment-31328724 .

Reply:
Fixed 39401ea. Left a comment there to explain the default.

Reply:
> Left a comment there to explain the default.

No need to explain the default. Please remove the comment. It is up to the reader's documentation to explain such details.